packages/resource-schemas/carbon-asset.schema.json

```diff
@@ -0,0 +1,202 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://carbondesignsystem.com/schemas/carbon-asset.schema.json",
```

Comment:
How does this `$id` get used? Since we aren't hosting these on cds.com, should use use the unpkg URLs?